src/config.c

```diff
@@ -407,6 +421,11 @@ RSConfigOptions RSGlobalConfigOptions = {
          .helpText = "interval in which to run the fork gc (relevant only when fork gc is used)",
          .setValue = setForkGcInterval,
          .getValue = getForkGcInterval},
+        {.name = "MAX_RESULTS_TO_UNSORTED_MODE",
```

Comment:
this configuration setting should be underscored, as it is not useful (and should not be touched) by the user, and should be commented "for testing purposes"